src/execution/execute.ts

```diff
@@ -164,7 +209,9 @@ export interface ExecutionArgs {
  * If the arguments to this function do not result in a legal execution context,
  * a GraphQLError will be thrown immediately explaining the invalid input.
  */
-export function execute(args: ExecutionArgs): PromiseOrValue<ExecutionResult> {
+export function execute(
+  args: ExecutionArgs,
+): PromiseOrValue<ExecutionResult | AsyncIterable<AsyncExecutionResult>> {
```

Comment:
Shouldn't this return be similar to the `subscribe` function return `AsyncGenerator< AsyncExecutionResult>`, instead of `AsyncIterable<AsyncExecutionResult>`, so that TypeScript consumers do not have to call `result.!()` or `result?.()` instead of just `return()` for disposing of the source if it is no longer consumed?

Reply:
Yes, just updated it